osutil/syscalls_linux_386.go

```diff
@@ -0,0 +1,4 @@
+package osutil
+
+// sysRenameAt2 is the number of the renameat2 system call.
```

Comment:
Should we include something like:
```
// Obtained via: scmp_sys_resolver -a x86 renameat2 
```
here too?

Reply:
I didn't obtain the numbers this way. I used the system call table published by hrw and cross-checked with systemd source code which has a similar piece of code.